package.json

```diff
@@ -1906,6 +1906,10 @@
             ]
         },
         "commands": [
+            {
+                "command": "github.copilot.generate",
+                "title": "Open CodeWhisperer Settings"
```

Comment:
Does this override `github.copilot.generate` completely, or only "hook" into it (without overriding their functionality)?

Reply:
It does not override github.copilot.generate completely, but it will change the text of it even in Copilot, so I am removing it in the latest commit